package.json

```diff
@@ -1,22 +1,31 @@
 {
   "name": "bee-queue",
-  "version": "0.3.0",
+  "version": "1.0.0",
   "description": "A simple, fast, robust job/task queue, backed by Redis.",
   "main": "index.js",
   "dependencies": {
-    "redis": "1.0.0"
+    "promise-callbacks": "^3.0.0",
```

Comment:
We prefer to not pin dependencies, because it seems more the job of the application to ensure that its dependencies are locked. In our case, we use `yarn`, but other folks may use npm shrinkwrap or lockfiles. For us, not pinning dependencies is beneficial because [yarn deduplicates common dependencies](https://gist.github.com/vvo/84a94cfc0f94c91ea6b6#pinning-dependencies-will-result-in-duplicated-code-and-modules-in-builds).